### PR TITLE
added support links to the admin ui

### PR DESCRIPTION
**Pull request by m-kharbat**
screen shot 2016-11-23 at 15 57 00

the image link will be fixed by pepper.swat.io

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - added support links to the admin ui.
```
Comment (chaudum) [inline CHANGES.txt]: more descriptive please, e.g.

> Added Pepper contact widget which displays various Crate.IO support channels in a user friendly way.

**Comment by chaudum** [comment]
so intercom support was actually already removed with this commit https://github.com/crate/crate-admin/pull/145

**Comment by celaus** [comment]
Intercom chat will be disabled from within Intercom :)

**Comment by m-kharbat** [comment]
intercom is removed from the admin ui, but maybe we should bring it back for surveys

**Pull request merged**